examples/minimal/client/rust/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+pub mod proto {
```

Comment:
Is there a reason for creating a separate library crate to load the generated code from the proto files? If other crates will not be depending on this, it is probably better to just include the generate files in main.rs, or move this into a submodule.

Reply:
The integration test needs this as it uses the client proto. This was not included in the initial change (and the test was failing because of this).